### PR TITLE
Chore:Move secret key to env variables

### PR DESCRIPTION
**Pull request by ThaDeveloper**
This PR solves this issue [#34](https://github.com/thenewboston-developers/Website-API/issues/34)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -51,3 +51,4 @@ jobs:
           POSTGRES_DB: thenewboston
           POSTGRES_USER: thenewboston
           POSTGRES_PASSWORD: thenewboston
+          SECRET_KEY: ${{ secrets.SECRET_KEY }}
```
Comment (vosi) [inline .github/workflows/main.yml:54]: remove that

File: config/settings/base.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-SECRET_KEY = 'g#$0(*8%8af27k7-e!ll^!-4yxomcx8ljv_o&_z*zhvi)f8&e7'
+SECRET_KEY = os.environ['SECRET_KEY']
```
Comment (vosi) [inline config/settings/base.py:6]: os.environ.get('SECRET_KEY', 'some random string')

**Review by vosi — CHANGES_REQUESTED** [review]

**Comment by vosi** [comment]
You should also add this to template.yml

File: .github/workflows/main.yml (hunk 1 above)
Comment (ThaDeveloper) [reply .github/workflows/main.yml:54]: How will we handle the failing tests?

Comment (vosi) [reply .github/workflows/main.yml:54]: with my comment below

**Comment by shashanksinha89** [comment]
> You should also add this to template.yml

then this will require  [ssm](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html#dynamic-references-ssm) or [secret-manager](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html#dynamic-references-secretsmanager) for fetching the secret.

**Comment by vosi** [comment]
@shashanksinha89 yeah, I can do that

File: template.yml [hunk 3]
```diff
@@ -277,6 +277,14 @@ Resources:
               - - '{{resolve:ssm:'
                 - !Sub "${EnvironmentName}SentryDsn"
                 - ':1}}'
+        - Namespace: aws:elasticbeanstalk:application:environment
```
Comment (vosi) [inline template.yml:280]: linebreak is missing here, keep the original formating

**Review by vosi — CHANGES_REQUESTED** [review]

File: template.yml (hunk 3 above)
Comment (ThaDeveloper) [reply template.yml:281]: done

**Review by shashanksinha89 — APPROVED** [review]

**Pull request merged**